libs/bower/README.md

```diff
@@ -0,0 +1,25 @@
+# How to add dependencies with bower in ILIAS
+
+**New dependencies need to be approved by the Jour Fixe of the ILIAS society or on behalf of 
+the Technical Board.**
```

Comment:
I currently don't understand what "approved ... on behalf of the Technical Board" should mean. Can someone approve new deps as a delegate for the TB? What is the use case for that?

I would suggest: **New dependencies need to be approved by the Jour Fixe of the ILIAS society or by the Technical Board.**